libs/schemas/src/queries/token.schemas.ts

```diff
@@ -2,12 +2,17 @@ import { z } from 'zod';
 import { Token } from '../entities';
 import { PaginatedResultSchema, PaginationParamsSchema } from './pagination';
 
+export const TokenView = Token.extend({
```

Comment:
Yep sorry about pushing that broken code. This will work, but you should add these to the main Token schema. We don't actually get the intial_supply or the launchpad_liquidity yet, as I am waiting on https://github.com/hicommonwealth/common-protocol/pull/181 to be finished and deployed to a testnet

Reply:
Ian deployed the contracts already. These are on the main schema but the schema defines them as `PG_ETH` which is `z.bigint()`. This gets serialized as a string when fetched from the DB. I added a note to discuss this in platform retro tomorrow. We either need to add explicit getters in the sequelize model to transform to `bigint` or change the schema to use string by default.